requirements.txt

```diff
@@ -1,3 +1,4 @@
 pymongo
 mysql-connector-python
 requests
+python-twitter
```

Comment:
While this is not an issue I have been thinking several times about splitting the dependencies apart a bit (dependencies for pastepwn to function and dependencies for certain analyzers/actions to function) -  see #38 for example. I am not a 100% sure on how to handle this, but this should be referenced, so that I think about it later.